File: problems/121-Best-Time-to-Buy-and-Sell-Stock/solution.c
int maxProfit(int* prices, int pricesSize){
    int maxPrice = 0;
    int maxProfit = 0;
    
    for(int i = pricesSize - 1; i >= 0; i --) {
        if(prices[i] > maxPrice) {
            maxPrice = prices[i];
        } else {
            maxProfit = maxProfit > maxPrice - prices[i] ? maxProfit : maxPrice - prices[i];
        }
    }
    
    return maxProfit;
}

File: problems/121-Best-Time-to-Buy-and-Sell-Stock/solution.go
func maxProfit(prices []int) int {
	Price, maxProfit := 10000, 0

	 i := 0; i < len(prices); i++ {
		if	prices[i] <= minPrice {
			minPrice = prices[i]
			continue
		}

		if prices[i]-minPrice > maxProfit {
			maxProfit = prices[i] - minPrice
		
	

	urn maxProfit
}

